### PR TITLE
Update default service type from LoadBalancer to NodePort and update nodeAffinity for filebeat/apiserver-audit-installer

### PR DESCRIPTION
**Pull request by wuchang0201**
1、update service from loadbalance to nodeport
2、update annfinity
kindest/node: v1.25.16
test report：
3181C357-DECF-4E61-A523-B16CC8FFDFA2
813123A4-2B12-4F69-AE09-D5A8CC2B3886

### CODE REVIEW

**Review by d3c3mber — COMMENTED** [review]
seems the readme file is missing, can you please add it here or maybe we can have the readme file in a separate PR?

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml
[the remark this answers is not in the document]
Comment (wuchang0201) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:1]: done

**Comment by wuchang0201** [comment]
add report:

**Review by d3c3mber — APPROVED** [review]

**Review by linuzb — APPROVED** [review]
LGTM

**Pull request merged**